src/styles/addForm.css

```diff
@@ -0,0 +1,15 @@
+.form {
+  width: 94%;
+}
```

Comment:
- [ ] Please, on form add a space between inputs and button. 🐦 
![image](https://user-images.githubusercontent.com/20563252/135684553-e29670e4-30cb-4ee0-92c5-3dced26b53f3.png)

- [ ] Try to make it bigger the inputs, more according the template.
![image](https://user-images.githubusercontent.com/20563252/135684627-b2f3727e-dc79-48f4-98d1-55a73208d1f2.png)